integration/run_test.go

```diff
@@ -332,3 +336,69 @@ func TestRunTailDeployment(t *testing.T) {
 		"Hello world! 2",
 	)
 }
+
+func TestRunTest(t *testing.T) {
+	MarkIntegrationTest(t, CanRunWithoutGcp)
+
+	tests := []struct {
+		description  string
+		testDir      string
+		testFile     string
+		args         []string
+		skipTests    bool
+		expectedText string
+	}{
+		{
+			description:  "Run test",
+			testDir:      "testdata/custom-test",
+			testFile:     "testdata/custom-test/runtest",
+			args:         []string{"--profile", "custom"},
+			skipTests:    false,
+			expectedText: "foo\n",
+		},
+		{
+			description:  "Run test with skip test false",
+			testDir:      "testdata/custom-test",
+			testFile:     "testdata/custom-test/runtest",
+			args:         []string{"--profile", "custom", "--skip-tests=false"},
+			skipTests:    false,
+			expectedText: "foo\n",
+		},
+		{
+			description: "Run test with skip test true",
+			testDir:     "testdata/custom-test",
+			testFile:    "testdata/custom-test/runtest",
+			args:        []string{"--profile", "custom", "--skip-tests=True"},
+			skipTests:   true,
+		},
+	}
+	for _, test := range tests {
+		t.Run(test.description, func(t *testing.T) {
+			defer func() {
+				os.Remove(test.testFile)
+			}()
```

Comment:
This can be simplified to `defer os.Remove(test.testFile)`

Reply:
Updated.